backend/src/tasks/functions.yml

```diff
@@ -58,3 +58,125 @@ updateScanTaskStatus:
           detail:
             clusterArn:
               - ${file(env.yml):${self:provider.stage}-ecs-cluster, ''}
+
+populateServiceStatsElasticache:
+  handler: src/xfd_django/xfd_api/tasks.populate_ServicesStatscache
```

Comment:
Make sure these paths are going to the correct functions. I don't see a tasks.populate...